fastlane/report.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need to keep this report in repo? It [seems](https://docs.fastlane.tools/best-practices/source-control/) like it can be added to .gitignore file.